pkgs/data/fonts/lilex/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenvNoCC, fetchzip }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "lilex-font";
+  version = "2.200";
+
+  src = fetchzip {
+    url = "https://github.com/mishamyrt/Lilex/releases/download/${version}/Lilex.zip";
+    sha256 = "sha256-MPQfqCMFMjcAlMos1o4bC+I+cvQYwr2TjI4Q03QeuaQ=";
+    stripRoot = false;
+  };
+
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm644 variable/*.ttf -t $out/share/fonts/lilex
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Open source programming font";
+    longDescription = ''
+      Lilex is the modern programming font containing a set of ligatures for
+      common programming multi-character combinations.
+    '';
+    homepage = "https://github.com/mishamyrt/Lilex";
+
+    license = licenses.ofl;
+    maintainers = with maintainers; [ chriscoffee ];
+    platforms = with platforms; all;
```

Comment:
```suggestion
    license = lib.licenses.ofl;
    maintainers = with lib.maintainers; [ chriscoffee ];
    platforms = lib.platforms.all;
```

Reply:
Updated with requested changes in https://github.com/NixOS/nixpkgs/pull/241666/commits/6c2d2a4a520866305c9c2b0b4334e72db90864ea